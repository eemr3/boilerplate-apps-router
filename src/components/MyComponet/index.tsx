import * as S from './styles';

const MyComponet = () => (
  <S.Wrapper>
    <h1>MyComponet</h1>
  </S.Wrapper>
);

export default MyComponet;
