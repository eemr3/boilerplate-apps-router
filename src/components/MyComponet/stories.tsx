import { Meta, StoryObj } from '@storybook/react';
import MyComponet from '.';

export default {
  title: 'MyComponet',
  component: MyComponet,
  parameters: {
    layout: 'fullscreen',
  },
} as Meta;

export const Default: StoryObj = {};
