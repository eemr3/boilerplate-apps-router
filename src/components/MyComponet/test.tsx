import { render, screen } from '@testing-library/react';

import MyComponet from '.';

describe('<MyComponet />', () => {
  it('should render the heading', () => {
    const { container } = render(<MyComponet />);

    expect(screen.getByRole('heading', { name: /MyComponet/i })).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
