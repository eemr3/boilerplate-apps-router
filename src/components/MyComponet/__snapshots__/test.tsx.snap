// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyComponet /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    MyComponet
  </h1>
</main>
`;
